temporal/api/cloud/namespace/v1/message.proto

```diff
@@ -0,0 +1,120 @@
+syntax = "proto3";
+
+package temporal.api.cloud.namespace.v1;
+
+option go_package = "go.temporal.io/api/cloud/namespace/v1;namespace";
+
+import "google/protobuf/timestamp.proto";
+
+message CertificateFilterSpec {
+    // The common_name in the certificate.
+    // Optional, default is empty.
+    string common_name = 1;
+    // The organization in the certificate.
+    // Optional, default is empty.
+    string organization = 2;
+    // The organizational_unit in the certificate.
+    // Optional, default is empty.
+    string organizational_unit = 3;
+    // The subject_alternative_name in the certificate.
+    // Optional, default is empty.
+    string subject_alternative_name = 4;
+}
+
+message MtlsAuthSpec {
+    // The base64 encoded ca cert(s) in PEM format that the clients can use for authentication and authorization.
+    // This must only be one value, but the CA can have a chain.
+    string accepted_client_ca = 1;
+    // Certificate filters which, if specified, only allow connections from client certificates whose distinguished name properties match at least one of the filters.
+    // This allows limiting access to specific end-entity certificates.
+    // Optional, default is empty.
+    repeated CertificateFilterSpec certificate_filters = 2;
+}
+
+message CodecServerSpec {
```

Comment:
```suggestion
message UICodecServerSpec {
```

Mentioned before but now that I think about it, a user may assume this codec server will be used for everything instead of just the UI. Not a strong opinion here though.

Reply:
I don't know.. its totally possible that non ui client can use the endpoint field... the other fields don't make a lot of sense though.